doc/source/contribution_and_api/unit_testing.rst

```diff
@@ -0,0 +1,42 @@
+.. _ref_unit_testing_contributing:
+
+Unit Testing
+===============
+
+Unit testing is highly important. It enables to check that the
+changes on the code are efficient and don't have an impact on
+other parts of the code. 
+
+Unit tests are in the ``tests`` directory inside this repository.
```

Comment:
Maybe you should explain here what the unit tests. "The unit tests are functions which, when run, checks that the output of some parts of code are valid". Double check this made up definition.

Reply:
Unit tests are testing functions which validate that the software logic implemented inside a certain method, class or module is working as expected. They should try to be as atomic and independent as possible.

However, PyAnsys does not do "unit" tests... they are more of "integration" tests honestly... I would also explain this whole dilemma here.